.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,7 @@
 
 TODO:
 
-- [ ] Update CHANGELOG.md with your change (include reference to issue & this PR)
+- [ ] Commits are formatted for [Standard Version](https://github.com/bcoe/conventional-changelog-standard/blob/master/convention.md)
```

Comment:
I agree with Sashko, we should put more instructions at the bottom of the PR template, and the examples should be taken from earlier commits in this repo that weren't written that way.

I also think it would be good, if we could update README.md to say that we're using standard-version and why.